app/waf/src/controllers/adc.controller.ts

```diff
@@ -641,3 +629,133 @@ type AddonReqValues = {
   userToken: string;
   tenantId: string;
 };
+
+class AdcStateCtrlr {
+  private defaultCheck = (): Promise<boolean> => {
+    return Promise.resolve(true);
+  };
+  private states: AdcStateEntry[] = [
+    {
+      state: AdcState.NEW,
+      check: Promise.resolve(true),
```

Comment:
did you intend to use this.defaultCheck() here?

Reply:
Planned to, but in fact, no use case found. 
Done to remove `defaultCheck`, will add it back if necessary.

Reply:
👌， removal seems fine.